muffnn/mlp/tests/test_base.py

```diff
@@ -10,7 +10,7 @@
 import muffnn.mlp.base as base
 
 
-class TestEstimator(base.MLPBaseEstimator):
```

Comment:
So calling this `TestEstimator` causes pytest to throw a warning, which is why I renamed it.